packages/calypso-build/babel.config.js

```diff
@@ -1,9 +1,18 @@
+const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
+const isBrowser = isCalypsoClient || 'true' === process.env.TARGET_BROWSER;
+
+const modules = isBrowser ? false : 'commonjs'; // Use commonjs for Node
+
+// Use target configuration in package.json for browser builds.
```

Comment:
>in package.json

Could also be other files or env var at consumer's side: https://github.com/browserslist/browserslist#packagejson

Would this be clear while less specific:

> Inherit Browserslist configuration for browser builds.
> https://github.com/browserslist/browserslist